### DISCUSSION TITLE
此链接的脚本支持添加为v2的组件，请问该如何操作

### DISCUSSION THREAD

**Discussion by awsl2546** [the1812/Bilibili-Evolved#3114] [提问 / Q&A] [answered]
https://greasyfork.org/zh-CN/scripts/421050-bilibili-%E6%98%BE%E7%A4%BA%E8%A7%86%E9%A2%91%E5%88%86%E5%8C%BA-av%E5%8F%B7%E7%AD%89
![QQ截图20220312214156](https://user-images.githubusercontent.com/90880106/158020344-909d1d3a-af0c-4bcb-9d06-373d24dced7e.png)

**Comment by the1812** [comment, marked as answer]
将他的安装链接复制, 在脚本的组件管理里粘贴安装 (首页有图示)

**Reply by awsl2546** [reply]
> 将他的安装链接复制, 在脚本的组件管理里粘贴安装 (首页有图示)

弄好了，感谢。
